coalib/bears/Bear.py

```diff
@@ -47,6 +48,25 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+    To indicate the requirements of the bear, give to the ``REQUIREMENTS``
+    tuple instances of the ``PackageRequirement`` class
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = (PackageRequirement('pip',
+    ...                                        'coala_decorators',
+    ...                                        '0.2.1'),)
+    If your bear uses requirements from a manager we have a subclass from,
+    you can use the subclass, such as ``PythonRequirement``, without specifying
+    manager:
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = (PythonRequirement('coala_decorators', '0.2.1'),)
+    Alternatively, you should use the ``multiple`` method if you have more
+    requirements from the same manager.
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = (PythonRequirement.multiple('coala_decorators',
+    ...                                                '0.2.1',
+    ...                                                'setuptools',
+    ...                                                '19.2'))
```

Comment:
or just providing `latest` as a version string is a special value that omits the version :)